src/api/procedures/types.ts

```diff
@@ -75,6 +79,85 @@ export type GetTransferRestrictionReturnType<T> = ActiveTransferRestrictions<
     : ClaimPercentageTransferRestriction
 >;
 
+export type RemoveAssetStatParams = { ticker: string } & (
+  | RemoveCountStatParams
+  | RemoveBalanceStatParams
+  | RemoveScopedCountParams
+  | RemoveScopedBalanceParams
+);
+
+export type AddCountStatParams = AddCountStatInput & {
+  type: StatType.Count;
+};
+
+export type AddBalanceStatParams = {
+  type: StatType.Percentage;
+};
+
+export type AddClaimCountStatParams = ClaimCountStatInput & {
+  type: StatType.ScopedCount;
+};
+
+export type AddClaimPercentageStatParams = StatClaimIssuer & {
```

Comment:
`AddBalanceStatParams` and `AddClaimPercentageStatParams` have inconsistent naming